packages/mobile/src/consumerIncentives/ConsumerIncentivesHomeScreen.tsx

```diff
@@ -98,92 +47,68 @@ export default function ConsumerIncentivesHomeScreen(props: Props) {
     })
   }
 
-  const onLearnMore = () => navigate(Screens.WebViewScreen, { uri: CELO_REWARDS_LINK })
+  const onLearnMore = () => navigateToURI(CELO_REWARDS_T_AND_C)
```

Comment:
Is there a reason we're doing `navigateToURI` instead of opening the uri in the `WebViewScreen` like we used to?

Reply:
oh I just personally like it when apps open links in my default browser, but I have changed it back